src/test/java/org/openstreetmap/atlas/geography/atlas/items/AtlasItemIntersectionTest.java

```diff
@@ -53,6 +53,22 @@ public void testAreasIntersectingPolygon()
                 Iterables.size(atlas.areasIntersecting(overlapping)));
     }
 
+    @Test
+    public void testAreasNoIntersectingPolygon()
+    {
+        final Atlas atlas = this.rule.getNoIntersectionAtlas();
+
+        final Polygon triangle = new Polygon(Location.forString("47.6263, -122.209198"),
+                Location.forString("47.628685, -122.209305"),
+                Location.forString("47.628704, -122.211761"));
+
+        Assert.assertEquals("There should be no intersecting area", 0,
+                Iterables.size(atlas.areasIntersecting(triangle)));
+
+        Assert.assertEquals("There should be no intersecting area", 0,
+                Iterables.size(atlas.areasIntersecting(triangle.bounds())));
```

Comment:
If I follow your diagram correctly, shouldn't this return an area? The triangle does not intersect anything, but it seems that its bounds do.

Reply:
Oh. Is that initial `triangle` the orange one?

Reply:
Yes